cmake/Modules/CMakeLists.txt

```diff
@@ -22,6 +22,20 @@ set(LANG_FILES
   CMakeTestCUDAQCompiler.cmake
 )
 
+execute_process(
+  COMMAND git rev-parse HEAD
+  WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}/tpls/fmt
+  OUTPUT_VARIABLE CUDAQ_FMTLIB_HASH
+  OUTPUT_STRIP_TRAILING_WHITESPACE
```

Comment:
I'm wondering if there is a way to do this that would work without the submodule being populated, i.e. perhaps something like `git submodule status tpls/fmt`. (Maybe your command already handles .... I don't really know.)

That being said, if it works in the CI, then it probably doesn't matter that much to me. So I'll kick off the CI.

Reply:
The git submodules are always initialized on the first invocation of CMake. 

https://github.com/NVIDIA/cuda-quantum/blob/main/CMakeLists.txt#L110